packages/pn-personafisica-webapp/src/__test__/test-utils.tsx

```diff
@@ -37,8 +39,51 @@ const axe = configureAxe({
 });
 expect.extend(toHaveNoViolations);
 
+/**
+ * Utility function to mock api response
+ * @param client Axios client
+ * @param method the api method: 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH'
+ * @param path the api path
+ * @param code the response code
+ * @param request body request
+ * @param response response
+ * @returns the mock instance
+ */
+function mockApi(
```

Comment:
@AndreaCimini90 do you think is possibile to put this mock function in pn-commons?